src/atoms/SubmitButton/SubmitButton.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./SubmitButton.css";
+
+const SubmitButton = ({ onClick }) => {
+  return <button className="submit-button" onClick={onClick}>Adicionar</button>;
```

Comment:
according to the idea of BEM, maybe the name button should come first in the class

Reply:
did you mean button-submit.    button-lie? @julialbq  @kabernardes

Reply:
In fact, it will be a button with two variations: submit and like

Reply:
Done